### PR TITLE
Add a page for unsupported operating systems

### PR DESCRIPTION
**Pull request by hatf0**

### CODE REVIEW

**Comment by cmiller01** [comment]
Link T-9379

File: references/cli-man-page/unsupported-os.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## 1. Download the tarball
+
+For the `stable` release track (this is most likely what you want), click the respective link for your CPU architecture.
+
+* [ARM64](https://get.devzero.io/stable/linux-arm64/dz.tar.gz) - less common (Raspberry Pi, etc)
+* [AMD64](https://get.devzero.io/stable/linux-amd64/dz.tar.gz) - more common
+
+If you're feeling adventurous, these links are for the `latest` release track, which has potentially unstable updates that are pushed more frequently.
+
+* [ARM64](https://get.devzero.io/latest/linux-arm64/dz.tar.gz) - less common (Raspberry Pi, etc)
```
Comment (cmiller01) [inline references/cli-man-page/unsupported-os.md:18]: wdyt about adding links for Darwin?

File: references/cli-man-page/unsupported-os.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Unsupported Operating Systems
```
Comment (cmiller01) [inline references/cli-man-page/unsupported-os.md:1]:
```suggestion
# Alternative Installation Steps
```
it's not exactly "unsupported" if we support it, it's more like a different way to install...WDYT?

Comment (hatf0) [reply references/cli-man-page/unsupported-os.md:1]: hmm, fair... let me rename this then

File: references/cli-man-page/unsupported-os.md (hunk 1 above)
Comment (hatf0) [reply references/cli-man-page/unsupported-os.md:18]: meh - what's the rationale behind that? no sudo access on darwin?

Comment (cmiller01) [reply references/cli-man-page/unsupported-os.md]: Like I want to just install the binary, or yea perhaps the no sudo access

File: references/cli-man-page/alternative-install.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+{% code %}
+```
+sudo ./dz net connect
+```
+{% endcode %}
+
+## 6. Success
+
+You're connected! You can now connect to your workspaces via `dz workspace connect`. Have fun!
```
Comment (dray92) [inline references/cli-man-page/alternative-install.md:82]:
```suggestion
You're connected! You can now connect to your workspaces via `./dz workspace connect`. Have fun!
```

File: references/cli-man-page/alternative-install.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+* [Intel](https://get.devzero.io/stable/darwin-amd64/dz.tar.gz) - more common
+
+* [Apple Silicon](https://get.devzero.io/stable/darwin-arm64/dz.tar.gz) - less common (Raspberry Pi, etc)
+
+If you're feeling adventurous, these links are for the `latest` release track, which has potentially unstable updates that are pushed more frequently.
+
+* [Intel](https://get.devzero.io/latest/darwin-amd64/dz.tar.gz) - more common
+
+* [Apple Silicon](https://get.devzero.io/latest/darwin-arm64/dz.tar.gz) - less common (Raspberry Pi, etc)
```
Comment (dray92) [inline references/cli-man-page/alternative-install.md:40]:
```suggestion
* [Intel](https://get.devzero.io/stable/darwin-amd64/dz.tar.gz) - older

* [Apple Silicon](https://get.devzero.io/stable/darwin-arm64/dz.tar.gz) - newer

If you're feeling adventurous, these links are for the `latest` release track, which has potentially unstable updates that are pushed more frequently.

* [Intel](https://get.devzero.io/latest/darwin-amd64/dz.tar.gz) - older

* [Apple Silicon](https://get.devzero.io/latest/darwin-arm64/dz.tar.gz) - newer
```

**Review by dray92 — APPROVED** [review]

File: SUMMARY.md [hunk 5]
```diff
@@ -118,6 +118,7 @@
   * [Updating the CLI](references/cli-man-page/update.md)
   * [Uninstall the CLI](references/cli-man-page/uninstall.md)
   * [Supported OSs](references/cli-man-page/supported-os.md)
+  * [Alternative Install](alternative-install.md)
```
Comment (dray92) [inline SUMMARY.md:121]:
```suggestion
  * [Alternative Install](references/cli-man-page/alternative-install.md)
```

**Pull request merged**